meta/main.yml

```diff
@@ -18,6 +18,6 @@ galaxy_info:
   - name: Ubuntu
     versions:
     - all
-  categories:
+  galaxy_tags:
```

Comment:
Is that parameter work with ansible 2.4 and above ?

Reply:
yes, I fixed the warning and run tests from 2.4.* to 2.7.*